src/code_8006E9C0.c

```diff
@@ -240,7 +241,7 @@ void func_8006F008(void) {
         D_80165728 = -330;
         break;
     case COURSE_LUIGI_RACEWAY:
-        D_8018D220 = dma_textures(gTextureExhaust2, 0x4F4U, 0xC00);
+        D_8018D220 = (void *)dma_textures(gTextureExhaust2, 0x4F4U, 0xC00);
```

Comment:
I'm wondering if `D_8018D220` should be a u8* instead of `void*` type. If it's always a texture that it's applied to.

Reply:
Took a closer look. Does (u8 *) work here? I think it's more correct in this scenario.